polkadot/runtime/common/src/auctions/benchmarking.rs

```diff
@@ -0,0 +1,222 @@
+use super::{Pallet as Auctions, *};
```

Comment:
These files need the copyright header.

Also a short, obvious description.

Finally, we should feature gate this file at this file level, so:

```
#![cfg(feature = "runtime-benchmarks")]
```

see examples in the regular substrate pallets:

<img width="1169" alt="image" src="https://github.com/user-attachments/assets/61057851-e163-4d5e-916b-1a1eb49cb2fe">

Reply:
Awesome will do that thx Shawn
I didnt add the "#![cfg(feature = "runtime-benchmarks")]" because i thought it was only for V2 benchmarking